client/src/components/add-job-lead-component/jobLeadCard.jsx

```diff
@@ -9,22 +10,61 @@ import {
   OutlinedInput,
   InputAdornment,
   FormHelperText,
+  Autocomplete,
 } from "@mui/material";
 import { AdapterDayjs } from "@mui/x-date-pickers/AdapterDayjs";
 import { LocalizationProvider, DatePicker } from "@mui/x-date-pickers";
 import dayjs from "dayjs";
 import DeleteIcon from "@mui/icons-material/Delete";
 import { JobLeadContainer, H3 } from "./index.styles";
-import EmployerType from "../../prop-types/EmployerType";
 import { JOB_TYPES } from "../../utils/contants";
+import { getFilteredEmployers } from "../../utils/api";
+
+function JobLeadContent({ jobLeadData, handleInputChange, handleDeleteJobLead, isAddEmployer }) {
+  const [open, setOpen] = useState(false);
+  const [options, setOptions] = useState([]);
+  const [error, setError] = useState(null);
+
+  const handleSearch = async (searchTerm) => {
+    if (searchTerm.length > 3) {
```

Comment:
I see that you are trying to restrict the number of results that can be returned by only showing employers once the search term name is > 3, however this means that any employer with a name <=3 will not be shown. I don't know what a good alternative would be but I'm happy to brainstorm.